sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateIssuer.cs

```diff
@@ -50,42 +50,42 @@ public CertificateIssuer(string name)
         }
 
         /// <summary>
-        /// The unique identifier of the certificate issuer
+        /// Gets the unique identifier of the certificate issuer.
         /// </summary>
         public Uri Id => Properties.Id;
 
         /// <summary>
-        /// The name of the certificate issuer
+        /// Gets the name of the certificate issuer.
         /// </summary>
         public string Name => Properties.Name;
 
         /// <summary>
-        /// The account identifier or username used to authenticate to the certificate issuer
+        /// Gets or sets the account identifier or username used to authenticate to the certificate issuer.
         /// </summary>
         public string AccountId { get; set; }
 
         /// <summary>
-        /// The password or key used to authenticate to the certificate issuer
+        /// Gets or sets the password or key used to authenticate to the certificate issuer.
         /// </summary>
         public string Password { get; set; }
 
         /// <summary>
-        /// The organizational identifier for the issuer
+        /// Gets or sets the organizational identifier for the issuer.
         /// </summary>
         public string OrganizationId { get; set; }
 
         /// <summary>
-        /// A list of contacts who administrate the certificate issuer account
+        /// Gets a list of contacts who administrate the certificate issuer account.
```

Comment:
"who administrate" ?  I realize administrators is redundant but it's a much more familiar word.